### DISCUSSION TITLE
2,69 chia dissappered from my wallet!

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#8578] [General]
2,69 chia dissappered from my wallet!
“Transaction has successfully been sent to a full node and included in the mempool.”
Outgoing xch1xrv5tu5pqzd5zsk38ftfm5qw0dpuw9eq7uq5x682annls649axaqd95x4r
09/24/2021 05:50:21
Confirmed at height 902381
2,69
0,0004

**Comment by loppefaaret** [comment]
if you are seeing that message in the client, the transaction was made on that machine!
does anyone else have physical access to the machine ? are you running any remote control software ?

**Comment by BrandtH22** [comment]
@elmans1961 , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopenning this ticket.
